src/provider/view/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$accent-color: #9391ff;
+$accent-color-highlight: lighten(saturate($accent-color, 40%), 10%);
```

Comment:
Does this duplicate the theme? What's the difference between these variables and the ones in properties.scss?

Reply:
Will address separately.